corral/bundle/json.pony

```diff
@@ -8,13 +8,13 @@ primitive Json
   fun load_object(file_path: FilePath, log: Log)
     : (JsonObject | FileErrNo | JsonError)
   =>
-    log.fine("Reading " + file_path.path)
+    //log.fine("Reading json from: " + file_path.path)
```

Comment:
why is the logging commented out?

Reply:
Even at 'fine', it was a bit noisy. Maybe I need a 'finer' level. I have added a new issue that I will work on next, to clean up the logging and user output.

Reply:
ok, that is reasonable for a cleanup of logging.